docs/java-rest/high-level/rollup/delete_job.asciidoc

```diff
@@ -0,0 +1,68 @@
+[[java-rest-high-x-pack-rollup-delete-job]]
+=== Delete Rollup Job API
+
+The Delete Rollup Job API can be used to create a new Rollup job
+in the cluster. The API accepts a `PutRollupJobRequest` object
+as a request and returns a `PutRollupJobResponse`.
```

Comment:
We now have a mecanism to avoid repeating this section, can you use it? (with DeleteRollupJobRequest instead of PutRollupJobRequest)

Reply:
I'd like to do this in a follow up PR: because there is a lot of re-use of `RollupDocumentationIT` across these, I'd like to just move (get|put|delete) job all at once, and adding that stuff to this PR doesn't feel right to me. I'll open the PR as soon as this is merged. seem reasonable?